packages/nitro-protocol/contracts/interfaces/IMultiAssetHolder.sol

```diff
@@ -40,6 +40,37 @@ interface IMultiAssetHolder {
         uint256[] memory indices
     ) external;
 
+    /**
+     * @param assetIndex Will be used to slice the outcome into a single asset outcome.
+     * @param sourceChannelId Unique identifier for a guarantor state channel.
+     * @param sourceStateHash Hash of the state stored when the guarantor channel finalized.
+     * @param sourceOutcomeBytes The abi.encode of guarantor channel outcome
+     * @param sourceAssetIndex the index of the targetted asset in the source outcome.
+     * @param indexOfTargetInSource The index of the guarantee allocation to the target channel in the source outcome.
+     * @param targetStateHash Hash of the state stored when the target channel finalized.
+     * @param targetOutcomeBytes The abi.encode of target channel outcome
+     * @param targetAssetIndex the index of the targetted asset in the target outcome.
+     * @param indices Array with each entry denoting the index of a destination (in the target channel) to transfer funds to. Should be in increasing order. An empty array indicates "all"
+     */
+    struct ClaimArgs {
+        bytes32 sourceChannelId;
+        bytes32 sourceStateHash;
+        bytes sourceOutcomeBytes;
+        uint256 sourceAssetIndex;
+        uint256 indexOfTargetInSource;
+        bytes32 targetStateHash;
+        bytes targetOutcomeBytes;
+        uint256 targetAssetIndex;
+        uint256[] indices;
```

Comment:
Earmark for #3728. Lots of arrays exist in implementation of `claim`, so what's being indexed isn't necessarily obvious.

Maybe a rename to `payeeIndices`?